deploy/manager/manager.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: governance-policy-status-sync
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: governance-policy-status-sync
+  template:
+    metadata:
+      labels:
+        name: governance-policy-status-sync
+        app: policy-framework
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      serviceAccountName: governance-policy-status-sync
+      containers:
+        - name: governance-policy-status-sync
+          image: quay.io/open-cluster-management/governance-policy-status-sync:latest
```

Comment:
Should this be -io?

Reply:
Images from this (`-io`) repo are pushed to quay.io/open-cluster-management, there is not a separate org.

Although, the image tag built here is `edge`, not `latest`, so that should be adjusted.